apps/user-portal/src/api/profile.ts

```diff
@@ -84,22 +84,25 @@ export const getProfileInfo = (): Promise<BasicProfileInterface> => {
             }
             if (isEmpty(response.data.userImage)) {
                 try {
-                    gravatar = await getGravatarImage(response.data.emails[0]);
+                    gravatar = await getGravatarImage(
+                        typeof response.data.emails[0] === "string"
+                            ? response.data.emails[0]
+                            : response.data.emails[0].value
+                    );
                 } catch (error) {
                     gravatar = "";
                 }
             }
             const profileResponse: BasicProfileInterface = {
-                displayName: response.data.name.givenName || "",
                 emails: response.data.emails || "",
-                lastName: response.data.name.familyName || "",
+                name: response.data.name || { givenName: "", familyName: "" },
                 organisation: response.data[orgKey] ? response.data[orgKey].organization : "",
                 phoneNumbers: response.data.phoneNumbers || [],
-                proUrl: response.data.profileUrl || "",
+                profileUrl: response.data.profileUrl || "",
                 responseStatus: response.status || null,
                 roles: response.data.roles || [],
-                userimage: response.data.userImage || gravatar,
-                username: response.data.userName || ""
+                userName: response.data.userName || "",
+                userimage: response.data.userImage || gravatar
```

Comment:
```suggestion
                userImage: response.data.userImage || gravatar
```

Reply:
@thivi - Please update the PR

Reply:
Merging the PR, let's fix in the next PR

Reply:
We could have add the code suggestion, without need of the PR author right?